.github/workflows/measure-test-flakiness.yaml

```
@@ -12,6 +12,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
-    - run: "./scripts/measure-test-flakiness.sh"
-      env:
+    - env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        ./scripts/measure-test-flakiness.sh
+        GOARCH=amd64 ./scripts/build_etcd_test_analyzer.sh
```

```suggestion
        make bin/etcd-test-analyser
```
If we are adding Makefile target for local use, let's use it in workflow. This way we test Makefile instead of letting it rot.

---

done. Thanks for suggestion.